spec/classes/init_spec.rb

```diff
@@ -196,6 +196,12 @@
               }
             else
               it {
+                # Stub /tmp_docker dir to prevent shelling out during spec test
```

Comment:
My guess is that these are hiding real errors.

Reply:
This stub/mock is to address the spec test only, not acceptance tests (which passed)